src/main/java/org/folio/converter/QuickMarcToParsedRecordConverter.java

```diff
@@ -58,56 +62,47 @@ private Record quickMarcJsonToMarcRecord(QuickMarcJson quickMarcJson){
         DataField dataField = factory.newDataField();
         dataField.setTag(field.getTag());
         dataField.getSubfields().addAll(stringToSubfields(field.getContent().toString()));
-        dataField.setIndicator1((field.getIndicators().get(0) == null)? ' ': field.getIndicators().get(0).toString().charAt(0));
-        dataField.setIndicator2((field.getIndicators().get(1) == null)? ' ': field.getIndicators().get(1).toString().charAt(0));
+        dataField.setIndicator1((field.getIndicators().get(0) == null)? SPACE_CHARACTER: field.getIndicators().get(0).toString().charAt(0));
```

Comment:
Is it possible to create separate method for `(field.getIndicators().get(0) == null)? SPACE_CHARACTER: field.getIndicators().get(0).toString().charAt(0)` to avoid duplication?